qdarkstyle/__init__.py

```diff
@@ -249,12 +259,15 @@ def _load_stylesheet(qt_api='', palette=None):
     if palette is None:
         from qdarkstyle.dark import style_rc
         palette = DarkPalette
+        _set_global_paths('dark')
```

Comment:
Thanks! @juliangilbey I'll revise locally because I would like to revise the previous code too. Attributing the `palette` shouldn't be needed in the other cases because we already received the palette. Also, `set_global_paths` can go outside ifs using `palette.ID` as the argument. But don't mind, I'll check.